docs/getting-started-guides/docker-multinode/master.sh

```diff
@@ -26,8 +26,8 @@ if ( ! ps -ef | grep "/usr/bin/docker" | grep -v 'grep' &> /dev/null ); then
 fi
 
 # Make sure k8s version env is properly set
-K8S_VERSION=${K8S_VERSION:-"1.2.0-alpha.7"}
-ETCD_VERSION=${ETCD_VERSION:-"2.2.1"}
+K8S_VERSION=${K8S_VERSION:-"1.2.2"}
+ETCD_VERSION=${ETCD_VERSION:-"2.3.1"}
```

Comment:
@thockin how the customers may clarify what the etcd versions are available to use?